File: Cementerio.java
package Trabajo;

public class Cementerio {
    
    public String color;

    public String tamaño;

    public String ubicacion;

    public Cementerio(String col, String tam, String ubi)
    {
        establecerColor(col);
        establecerTamaño(tam);
        establecerUbicacion(ubi);
    }


    public String obtenerColor()
    {
        return color;
    }

    public void establecerColor(String col)
    {
        color = col;
    }

    public String obtenerTamaño()
    {
        return tamaño;
    }

    public void establecerTamaño(String tam)
    {
        tamaño=tam;
    }


    public String obtenerUbicacion()
    {
        return ubicacion;
    }
    public void establecerUbicacion(String ubi)
    {
        ubicacion = ubi;
    }
}

File: cementerio.php
<?php

namespace App\Model;

class Cementerio
{
    public $tamaño = "";
    public $tipo = "";
    public $ubicacion = "";

    //nombre
    public function __construct($tam)
    {
       $this->establecerTamaño();
    }

    public function obtenerTamaño()
    {
        return $this->tamaño;
    }

    public function establecerTamaño($tam)
    {
        $this->tamaño = $tam;
    }

    //tipo
    public function __construct($tip)
    {
       $this->establecerTipo();
    }

    public function obtenerTipo()
    {
        return $this->tipo;
    }

    public function establecerTipo($tip)
    {
        $this->tipo = $tip;
    }

    //ubicacion
    public function __construct($ubi)
    {
       $this->establecerUbicacion();
    }

    public function obtenerUbicacion()
    {
        return $this->ubicacion;
    }

    public function establecerUbicacion($ubi)
    {
        $this->ubicacion = $ubi;
    }
}